lib/sidekiq/cli.rb

```diff
@@ -162,9 +162,12 @@ def self.banner
       },
       'TTIN' => ->(cli) {
         Thread.list.each do |thread|
-          Sidekiq.logger.warn "Thread TID-#{(thread.object_id ^ ::Process.pid).to_s(36)} #{thread['sidekiq_label']}"
+          prefix = "Thread TID-#{(thread.object_id ^ ::Process.pid).to_s(36)}"
+          Sidekiq.logger.warn "#{prefix} #{thread['sidekiq_label']}"
           if thread.backtrace
-            Sidekiq.logger.warn thread.backtrace.join("\n")
+            thread.backtrace.each do |trace_line|
```

Comment:
I think there was some previous discussion about how it was useful to have the entire backtrace as part of one logger line, so they couldn't get sorted out of order. WDYT?

Reply:
Interesting! That would also solve the issues I described. But I wonder: are there some technical aspects to logfiles/logservers that make super-long lines a bad idea? I know that Graylog [kinda has a limit of 20k characters per line](https://stackoverflow.com/questions/37613042/graylog-gelf-max-field-size) but that should be plenty. The other downside is that it's not that human-readable as multi-line output.